.travis/build.sh

```diff
@@ -3,10 +3,12 @@
 scripts_dir=$(dirname $(readlink -e "${BASH_SOURCE[0]}"))
 source "$scripts_dir/common";
 
+pip install protobuf PyYAML
+
 [[ -n "$CAFFE2_VERSION" ]] || die "CAFFE2_VERSION not set"
 
 onnx_c2_dir="$PWD"
-pip install -e $onnx_c2_dir
+CC=gcc-5 CXX=g++-5 pip install -e $onnx_c2_dir
```

Comment:
why gcc5?

Reply:
@dzhulgakov pytorch gcc4.8 build is currently broken.
@anderspapitto you probably also need to configure travis to install gcc5

Reply:
yeah, I have modified the travis setup in this diff.

(specifically, it's aten which doesn't build with gcc4.8